tests/unittests/core/test_trial.py

```diff
@@ -208,6 +208,72 @@ def test_gradient_property(self, exp_config):
         assert t.gradient.value == [5, 3]
         tmp = exp_config[1][1]['results'].pop()
 
+    def test_constraints_property(self, exp_config):
+        """Tests the property for accessing constraints"""
+        # 0 result of type 'constraints' exist
+        tmp = exp_config[1][1]['results'].pop(1)
+        t = Trial(**exp_config[1][1])
```

Comment:
Unless it makes fail something that I'm not aware of, you could use a blank Trial to test the constraints only. It would be better if we can avoid depending on the global exp_config.

Reply:
I agree. I did it like that originally to mimics the rest of the module.